src/main/java/com/accantosystems/stratoss/vnfmdriver/web/etsi/GrantController.java

```diff
@@ -46,12 +46,12 @@ public GrantController(GrantService grantService) {
     public ResponseEntity<Grant> requestGrant(@RequestBody GrantRequest grantRequest) throws GrantRejectedException, GrantProviderException {
         logger.info("Received grant request:\n{}", grantRequest);
         UUID uuid = UUID.randomUUID();
-        LoggingUtils.logEnabledMDC(grantRequest != null ? grantRequest.toString() : null, MessageType.REQUEST, MessageDirection.RECEIVED, uuid.toString(),MediaType.APPLICATION_JSON.toString(), "https",getRequestProtocolMetaData(GRANT_LOCATION) ,null);
+        LoggingUtils.logEnabledMDC(grantRequest != null ? grantRequest.toString() : null, MessageType.REQUEST, MessageDirection.RECEIVED, uuid.toString(),MediaType.APPLICATION_JSON.toString(), "https",getRequestProtocolMetaData(GRANT_LOCATION) ,grantRequest.getVnfLcmOpOccId());
```

Comment:
Chance of getting nullpointerexception for grantRequest.getVnfLcmOpOccId().

Please add if condition and create local variables to store message and lcmOpOccId and use in the request and response logs.

Reply:
Fixed